src/tests/access/test_accesscontrol.cairo

```diff
@@ -448,27 +448,32 @@ fn test_default_admin_role_is_its_own_admin() {
 //
 
 fn assert_event_role_revoked(role: felt252, account: ContractAddress, sender: ContractAddress) {
-    let event = utils::pop_log::<RoleRevoked>(ZERO()).unwrap();
-    assert_eq!(event.role, role);
-    assert_eq!(event.account, account);
-    assert_eq!(event.sender, sender);
+    let event = utils::pop_log::<AccessControlComponent::Event>(ZERO()).unwrap();
+    let expected = AccessControlComponent::Event::RoleRevoked(
+        RoleRevoked { role: role, account: account, sender: sender }
```

Comment:
```suggestion
        RoleRevoked { role, account, sender }
```
If we agree on this, applies for the rest of the helpers as well.

Reply:
Ahh good call. Force of habit on my part